### PR TITLE
Move docstrings to Unicode stdlib module

### PR DESCRIPTION
**Pull request by nalimilan**
Functions need to be defined in Unicode module, as reexporting Base.Unicode
functions makes them appear with the latter prefix in the manual.

### CODE REVIEW

File: base/strings/substring.jl [hunk 1]
```diff
@@ -107,7 +107,7 @@ pointer(x::SubString{String}, i::Integer) = pointer(x.string) + x.offset + (i-1)
 Reverses a string. Technically, this function reverses the codepoints in a string and its
 main utility is for reversed-order string processing, especially for reversed
 regular-expression searches. See also [`reverseind`](@ref) to convert indices in `s` to
-indices in `reverse(s)` and vice-versa, and [`Unicode.graphemes`](@ref Base.Unicode.graphemes) to
+indices in `reverse(s)` and vice-versa, and `graphemes` from module `Unicode` to
```
Comment (nalimilan) [inline base/strings/substring.jl]: I couldn't get the link to work, suggestions welcome.

Comment (KristofferC) [reply base/strings/substring.jl]: Did ```[`Unicode.graphemes`](@ref)```not work?

Comment (nalimilan) [reply base/strings/substring.jl]: No.

**Comment by nalimilan** [comment]
Bump.

File: base/strings/substring.jl (hunk 1 above)
Comment (KristofferC) [reply base/strings/substring.jl]: I think it is fine to leave it like this then until we figure it out,

Comment (nalimilan) [reply base/strings/substring.jl]: Yeah, anyway I don't have any solution to suggest. Merge then?

Comment (KristofferC) [reply base/strings/substring.jl]: Sure, perhaps run docs locally and see if it looks OK? Otherwise, looks good to me.

Comment (nalimilan) [reply base/strings/substring.jl]: They look OK, just the link isn't there (obviously :-).

**Pull request merged**